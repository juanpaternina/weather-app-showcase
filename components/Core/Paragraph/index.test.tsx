import { render, screen } from '@testing-library/react-native';

const { Paragraph } = require('.');

describe('Paragraph', () => {
  test('It should render a Text component', () => {
    const { toJSON } = render(<Paragraph>Paragraph</Paragraph>);
    expect(toJSON()).toMatchSnapshot();
    const text = screen.getByText('Paragraph');
    expect(text).toBeDefined();
  });
});
