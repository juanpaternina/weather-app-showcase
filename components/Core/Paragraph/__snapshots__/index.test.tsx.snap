// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph It should render a Text component 1`] = `
<Text
  style={
    {
      "color": "black",
      "fontFamily": "regular",
      "fontSize": 16,
      "fontWeight": "normal",
    }
  }
>
  Paragraph
</Text>
`;
